furnace.c

```diff
@@ -200,10 +209,21 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
                         "                  \t\t\t\t basic\n"
                         "                  \t\t\t 0 - off\n"
                         "                  \t\t\t 1 - on\n"
-                        "log               \t\t prints names of turned on log options\n";
+                        "log               \t\t prints names of turned on log options\n"
+                        "pulse <0:127>     \t\t starts pulses of magnetron\n";
```

Comment:
We probably should guard this too with CONFIG_MAGNETRON, right?

Reply:
I added this as a part of other commit as well as guarding other unused help commands.